src/closed.jl

```diff
@@ -0,0 +1,54 @@
+"""
+Defines the ClosedSet type.
+
+A ClosedSet(left, right) is an interval set where both bounds are closed i.e. the range is [left, right]
```

Comment:
I had the same concern as @timholy mentioned. But I guess the consensus is to use `ClosedIntervals` .